src/components/Temp/Temp.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {View} from 'react-native';
+import useLocalize from '../../hooks/useLocalize';
+import useWindowDimensions from '../../hooks/useWindowDimensions';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import variables from '../../styles/variables';
+import BlockingView from '../BlockingViews/BlockingView';
+import * as Illustrations from '../Icon/Illustrations';
+
+const propTypes = {
+    /** Child elements */
+    children: PropTypes.node,
+
+    /** If true, child components are replaced with a blocking "not found" view */
```

Comment:
If `false` blocking view will be still visible, because we have OR condition

Reply:
@ArekChr Updated the conditional can you verify? if the `shouldShow` is a non-boolean value we use `isSmallScreenWidth` as default else we follow `shouldShow` prop.

> `shouldShow` prop will be null/undefined as this is an optional param.